config/initializers/active_model_serializers.rb

```diff
@@ -1,7 +0,0 @@
-ActiveModel::Serializer.root = false
```

Comment:
because it was here only because 0.9. It's not even compatible with 0.8, and everything seems to work without it